lib/runner/variant-reassignment.js

```diff
@@ -71,17 +73,31 @@ export default class VariantReassignment {
     return Promise.resolve()
   }
 
-  async _handleProcessingError (productDraft, products) {
+  /**
+   * For 400 errors, we don't repeat the actions as the request itself is wrong.
+   * For other errors, we retry the actions.
```

Comment:
Specify other errors. Do you retry on 404 too?

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/61/commits/69d76dc806f152fb31e658a91276d0463a402331

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/61/commits/260d448b82ee9d5a0484360613f81d27f77672f3